src/m365/spp/commands/model/model-remove.spec.ts

```diff
@@ -0,0 +1,191 @@
+    sinon.stub(session, 'getId').returns('');
+    sinon.stub(spp, 'assertSiteIsContentCenter').resolves();
+    auth.connection.active = true;
+    commandInfo = cli.getCommandInfo(command);
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: async (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: async (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: async (msg: string) => {
+        log.push(msg);
+      }
+    };
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get,
+      request.delete,
+      cli.promptForConfirmation
+    ]);
+  });
+
+  after(() => {
+    sinon.restore();
+    auth.connection.active = false;
+  });
+
+  it('has correct name', () => {
+    assert.strictEqual(command.name, commands.MODEL_REMOVE);
+  });
+
+  it('has a description', () => {
+    assert.notStrictEqual(command.description, null);
+  });
+
+  it('passes validation when required parameters are valid with id', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoint.com/sites/sales', id: '9b1b1e42-794b-4c71-93ac-5ed92488b67f' } }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('passes validation when required parameters are valid with title', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoint.com/sites/sales', title: 'ModelName' } }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('passes validation when required parameters are valid with id and force', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoint.com/sites/sales', id: '9b1b1e42-794b-4c71-93ac-5ed92488b67f', force: true } }, commandInfo);
+    assert.strictEqual(actual, true);
+  });
+
+  it('fails validation when siteUrl is not valid', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'invalidUrl', id: '9b1b1e42-794b-4c71-93ac-5ed92488b67f' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation when id is not valid', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoint.com/sites/sales', id: 'foo' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('correctly handles an error when the model title is not found', async () => {
+    sinon.stub(request, 'delete').callsFake(async (opts) => {
+      if (opts.url === `https://contoso.sharepoint.com/sites/portal/_api/machinelearning/models/getbyuniqueid('9b1b1e42-794b-4c71-93ac-5ed92488b67f')`) {
+        throw {
+          error: {
+            "odata.error": {
+              code: "-1, Microsoft.Office.Server.ContentCenter.ModelNotFoundException",
+              message: {
+                lang: "en-US",
+                value: "File Not Found."
+              }
+            }
+          }
+        };
+      }
+    });
```

Comment:
When you remove a model by title, the response succeeds and the response body is `odata.null: true`. Let's check for this as well in the command and write a test for it.